activejob/test/cases/argument_serialization_test.rb

```diff
@@ -5,6 +5,7 @@
 require "models/person"
 require "active_support/core_ext/hash/indifferent_access"
 require "jobs/kwargs_job"
+require "action_controller/metal/strong_parameters"
```

Comment:
I don't this will work on projects that only use `activejob`. We can probably assume its already loaded.

Reply:
I wonder what the proper way is to assume this class already loaded in `argument_serialization_test.rb` test if it has dependency 🤔. Thanks

Some tests failed at
`uninitialized constant ArgumentSerializationTest::ActionController`

Reply:
@gmcgibbon 😜😜😜

Reply:
sounds like a good approach, thanks!

Reply:
Hi, @gmcgibbon 

I've followed the folder structure here and add a stub class for the `ActiveJob` test suite in order to remove the dependency
https://github.com/rails/rails/blob/master/actioncable/test/stubs/global_id.rb

You can take a look, thanks~